submissions/sprint_a/__init__.py

```diff
@@ -0,0 +1,12 @@
+def configure_app():
+    import os
```

Comment:
Missing function docstring describing purpose of function.
- [Description](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)
- [Example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)